doc/content/getting_started/installation.md

```diff
@@ -0,0 +1,74 @@
+# Install Moltres
+
+## 1. Install the Conda MOOSE Environment
+
+Moltres relies on the MOOSE framework. We suggest that users install the MOOSE environment using
+Conda packages by following the instructions in the "Install Mambaforge3" and "Install MOOSE Conda
+Packages" section of the MOOSE
+[installation guide](https://mooseframework.inl.gov/getting_started/installation/conda.html). The
+`moose-tools` and `moose-libmesh` Conda package versions which are compatible with the current
+MOOSE Git submodule on this repository are `moose-tools=2021.07.14` and `moose-libmesh=2021.07.14`.
+We cannot guarantee compatibility with the older/newer MOOSE builds and their corresponding Conda
+environments. Run the following command to install the specified package versions:
+
+```bash
+mamba install moose-tools=2021.07.14 moose-libmesh=2021.07.14
+```
+
+## 2. Clone Moltres
+
+The Moltres GitHub repository already contains MOOSE and Squirrel as Git submodules, therefore
++there is no need to clone and compile MOOSE separately+. 
+
+Abiding by MOOSE's instructions for setting up a `projects` directory for MOOSE-related content,
+users can clone Moltres and download MOOSE as a submodule by running the commands:
+
+```bash
+mkdir ~/projects
+cd ~/projects
+git clone https://github.com/arfc/moltres.git
+cd moltres
+git checkout master
+git submodule init
+git submodule update
+```
+
+## 3. Compile and Test Moltres
+
+Before proceeding with this step, check that the Conda MOOSE environment is activated in your
+current terminal. This is achieved by running the following command every time you open a new
+terminal:
+
+```bash
+conda activate moose
+```
+
+Finally, compile Moltres and run tests to check your installation using the following commands:
+
+```bash
+make -j8
+./run_tests -j8
+```
+
+!alert note
+You may also compile a debug version of Moltres by replacing the last line in 
+the code block above with `METHOD=dbg make
+-j8`. Note that you should replace `8` with the number of processors available
+on your machine.
+
+## 4. Running Moltres Simulations
+
+If you have successfully compiled Moltres, you should have a `moltres-opt` executable in the
+`~\projects\moltres` directory. To run your input file, use the following command:
```

Comment:
Without running an input, is there a command line way (e.g. `which` or `--version` to verify the installation works?

Reply:
This sentence may be redundant because the previous step "Compile and Test Moltres" serves as the verification step for whether the installation works. I think my intention here was to inform users where they can find the Moltres executable. I'll replace this sentence by simply stating that the installation creates a `moltres-opt` executable in the `~/projects/moltres` directory.